lua/cli_flash.lua

```diff
@@ -481,6 +481,10 @@ local tParserCommandCheckHelperVersion = tParser:command(
 addSecureArgs(tParserCommandCheckHelperVersion)
 
 -- check_helper_signature
+local strUsipHelp = [[
```

Comment:
Sorry, I forgot to add a help text here. It should be 
Verify if the signed helper binaries pass the signature check on the connected netX 90 device.